src/TSMapEditor/Models/Waypoint.cs

```diff
@@ -1,15 +1,63 @@
 ﻿using Rampastring.Tools;
 using TSMapEditor.GameMath;
+using Microsoft.Xna.Framework;
+using System;
+using System.Globalization;
 
 namespace TSMapEditor.Models
 {
     public class Waypoint : AbstractObject, IMovable
     {
+        public static NamedColor[] SupportedColors = new NamedColor[]
+{
```

Comment:
Incorrect horizontal bracket positioning

```suggestion
        {
```